src/src/AppContext.js

```diff
@@ -53,16 +55,19 @@ function AppProvider({ children }) {
     [selfServiceApiClient],
   );
 
-  const { addCapability } = useCapabilities();
-  const { profileInfo, isLoadedProfile } = useProfile(user);
+  const { capabilities, addCapability } = useCapabilities();
+  const { profileInfo, isLoadedProfile } = useProfile(user, reload);
   const { statsInfo, isLoadedStats } = useStats(user);
 
   async function addNewCapability(name, description, invitations) {
     addCapability(name, description, invitations);
-    await sleep(3000);
-    await loadMyProfile();
+    await sleep(1000); // purely here as it _feels_ better -- it is not strictly required
   }
 
+  useEffect(() => {
+    setReload(!reload);
+  }, [capabilities]);
```

Comment:
can you explain this part to me? When capabilities have been changed we flip the boolean, which triggers another hook?